examples/notebooks/Transpose.ipynb

```diff
@@ -165,14 +165,14 @@
     "    @assert w % BLOCK_SIZE == 0\n",
     "    @assert h % BLOCK_SIZE == 0\n",
     "    lmem = cl.LocalMem(Float32, BLOCK_SIZE * (BLOCK_SIZE + 1))\n",
-    "    clcall(k, Tuple{Ptr{T}, Ptr{T}, Cuint, Cuint, Ptr{Float32}}, dst, src, h, w, lmem;\n",
+    "    clcall(k, Tuple{Ptr{T}, Ptr{T}, Cuint, Cuint, cl.LocalMem{Float32}}, dst, src, h, w, lmem;\n",
```

Comment:
This was needed to fix `MethodError: no method matching unsafe_convert(::Type{Ptr{Float32}}, ::OpenCL.cl.LocalMem{Float32})`